### ISSUE TITLE
Runtime Catalogue: replace persistence-manager by storage-manager

### ISSUE THREAD

**Issue by pchainho** [reTHINK-project/dev-service-framework#66]
As agreed [here](https://github.com/reTHINK-project/dev-runtime-core/issues/119)

New release is pending on this one.

**Comment by jboulmal** [comment]
Please @Endebert update dist/  files as well.

**Comment by Endebert** [comment]
Just pushed a first version of the storageManager usage in RuntimeCatalogue to the develop branch: https://github.com/reTHINK-project/dev-service-framework/commit/1cbf2aaf3a2210f11788ee3e5382b8686e06a4f4

It's untested, since I do not have a test environment with the browser runtime using the master branch at hand. Please tell me how quickly setup such an environment, or, if someone has a matching environment already set up, please test my code and provide feedback.

I believe the code *should* work as long as my assumptions regarding the storageManager are correct.

**Comment by jboulmal** [comment]
All right @Endebert I can't test it from runtime-nodejs. However, in we need an updated version in the dist folder.
```

import { RuntimeCatalogueLocal, RuntimeCatalogue } from 'service-framework/dist/RuntimeCatalogue';
import storageManager  from 'service-framework/dist/storageManager';

```

**Comment by Endebert** [comment]
@jboulmal I have updated the dist folder, maybe you are using the master branch instead of develop?

Commit: https://github.com/reTHINK-project/dev-service-framework/commit/6825bdd1de2e286cb2cd8bd593f18f0a7ba3c52b

**Comment by jboulmal** [comment]
in runtime-nodejs we have only used the develop branch so far.

**Comment by Endebert** [comment]
Ok, but I pushed the updated dist folder commit together with the updated RuntimeCatalogue commit. Maybe delete the service framework folder and rerun npm install?

**Comment by jboulmal** [comment]
OK @Endebert!

**Comment by jboulmal** [comment]
I saw that you have kept the same name `PersistanceManager`.
On runtime-nodejs we got the following error message:
![screenshot from 2016-10-25 15 16 29](https://cloud.githubusercontent.com/assets/15377466/19687577/36e0b350-9ac6-11e6-8ed9-aec712b855d6.png)

**Comment by Endebert** [comment]
This is because the [nodejs RuntimeFactory](https://github.com/reTHINK-project/dev-runtime-nodejs/blob/master/src/RuntimeFactory.js) does not provide a `storageManager()` function, like the [browser runtime](https://github.com/reTHINK-project/dev-runtime-browser/blob/master/src/RuntimeFactory.js) does. The RuntimeCatalogue gets access to the storageManager this way, so it needs it to work.

**Comment by Endebert** [comment]
changes pushed into new branch, see here: https://github.com/reTHINK-project/dev-service-framework/blob/develop-catalogue-storagemanager/src/runtime-catalogue/RuntimeCatalogue.js

**Comment by pchainho** [comment]
what is the status of this one?

**Comment by jboulmal** [comment]
The integration with runtime-nodejs is progressing but we get following error : 
```
Node http Request
error is  TypeError: runtimeFactory.persistenceManager is not a function
    at new RuntimeUA (null:15108:47)
    at ~/dev-runtime-nodejs/dist/core.js:78:21
    at ~/dev-runtime-nodejs/dist/core.js:97:7
    at run (~/dev-runtime-nodejs/node_modules/IndexedDBShim/dist/indexeddbshim-node.js:3942:22)
    at~/dev-runtime-nodejs/node_modules/IndexedDBShim/dist/indexeddbshim-node.js:3955:28
    at flush (~/dev-runtime-nodejs/node_modules/IndexedDBShim/dist/indexeddbshim-node.js:1240:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by Endebert** [comment]
No further comments. Changes were merged into the develop branch. Closing issue

**Comment by jboulmal** [comment]
I have solved this small error by keeping also persistanceManager.
